lib/scala/project-manager/src/main/scala/org/enso/projectmanager/service/filesystem/FileSystemService.scala

```diff
@@ -21,23 +22,27 @@ class FileSystemService[F[+_, +_]: Applicative: CovariantFlatMap: ErrorChannel](
   projectRepositoryFactory: ProjectRepositoryFactory[F]
 ) extends FileSystemServiceApi[F] {
```

Comment:
Out of curiosity: `FileSystemServiceApi` what's that?
- there is [FileSystemServiceApi](https://github.com/enso-org/enso/blob/cc1ac87c9913ca4df29bc3fe07ae80e32dbac401/lib/scala/project-manager/src/main/scala/org/enso/projectmanager/service/filesystem/FileSystemServiceApi.scala#L5) in `lib/scala/project-manager`
- there is [FileSystem](https://github.com/enso-org/enso/blob/cc1ac87c9913ca4df29bc3fe07ae80e32dbac401/lib/scala/project-manager/src/main/scala/org/enso/projectmanager/infrastructure/file/FileSystem.scala#L9) in `lib/scala/project-manager`
- there is [FileSystem](https://github.com/enso-org/enso/blob/cc1ac87c9913ca4df29bc3fe07ae80e32dbac401/lib/scala/pkg/src/main/scala/org/enso/filesystem/FileSystem.scala#L21) in `lib/scala/pkg`
- there is [FileSystemApi](https://github.com/enso-org/enso/blob/cc1ac87c9913ca4df29bc3fe07ae80e32dbac401/engine/language-server/src/main/scala/org/enso/languageserver/filemanager/FileSystemApi.scala) in `language-server`
- there is [FileSystem](https://github.com/enso-org/enso/blob/cc1ac87c9913ca4df29bc3fe07ae80e32dbac401/engine/language-server/src/main/scala/org/enso/languageserver/filemanager/FileSystem.scala#L19) in `language-server`

Are we paid for duplicated lines of code or what?

Compare NetBeans [also has a FileSystem](https://bits.netbeans.org/dev/javadoc/org-openide-filesystems/org/openide/filesystems/FileSystem.html), but it is just once in the API and then used by all the other code.